test/TestUtilities/Test.Utility/FileSystemBackedV3MockServer.cs

```diff
@@ -158,7 +158,7 @@ private Action<HttpListenerResponse> ServerHandlerV3(HttpListenerRequest request
                         return new Action<HttpListenerResponse>(response =>
                         {
                             response.ContentType = "application/json";
-                            var vulnerabilityJson = FeedUtilities.CreateVulnerabilitiesJson(Uri + "/vulnerability/vulnerability.json");
+                            var vulnerabilityJson = FeedUtilities.CreateVulnerabilitiesJson(Uri + "vulnerability/vulnerability.json");
```

Comment:
Was this a bug?
Was it just a double `/`?

Reply:
yeah, the test was failing because the url had a double `/`. I honestly don't understand how other tests worked. Makes me wonder if other tests were even exercising this code path, but maybe the double `/` failure is OS specific.